editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -53,10 +53,11 @@ export const MembersTable = ({
   const [initialValues, setInitialValues] = useState<TeamMember | undefined>();
 
   const roleLabels: Record<Role, string> = {
-    platformAdmin: "Admin",
+    platformAdmin: "Platform admin",
+    teamAdmin: "Team admin",
     teamEditor: "Editor",
     teamViewer: "Viewer",
-    demoUser: "Demo User",
+    demoUser: "Demo user",
```

Comment:
`Demo User` to `Demo user` is a minor unrelated change for better consistency with other Planx roles and copy edit styles !